presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -70,6 +70,7 @@
     private BigDecimal levelTimeMultiplier = new BigDecimal(2.0);
 
     private boolean legacySchedulingBehavior = true;
+    private boolean pagesIndexEagerCompact = false;
```

Comment:
Ultimately, this isn't task manager's configuration. Is there a better place to put this?

btw, `= false` is default, should be omitted.

Reply:
I agree. But currently there is no better place to put this configuration (except creating a standalone config).

Thank you for the catch about default `false`~ :)

Reply:
yeah, this really doesn't belong here. `FeaturesConfig` maybe (it has a mix of feature-related stuff and low level stuff (like spiller threads etc)).